codec/src/main/java/io/netty/handler/codec/ByteToMessageDecoder.java

```diff
@@ -454,7 +453,7 @@ protected void callDecode(ChannelHandlerContext ctx, ByteBuf in, List<Object> ou
                     break;
                 }
 
-                if (outSize == out.size()) {
+                if (0 == out.size()) {
```

Comment:
We can now use `out.isEmpty()` instead which is more intuitive.

Reply:
> Thanks for the contribution @chenqwwq, this makes sense! Few minor comments and then LGTM

..sry click the wrong place